stix_shifter_modules/onelogin/README.md

```diff
@@ -0,0 +1,219 @@
+            "policy_name": "policy_name",
+            "otp_device_name": "Default",
+            "operation_name": "Default",
+            "directory_sync_run_id": "Default",
+            "directory_id": 12345678,
+            "resolution": "resolution",
+            "client_id": 12345678,
+            "resource_type_id": "Default",
+            "error_description": "error_description",
+            "proxy_ip": "121.0.0.1",
+            "risk_score": 2,
+            "risk_reasons": "risk_reasons",
+            "risk_cookie_id": 123,
+            "browser_fingerprint": true
+        }]'`
+
+Will return the following STIX observable:
+
+```json
+{
+    "type": "bundle",
+    "id": "bundle--a7a80b78-c022-4ba5-9540-c4ba3f5d782b",
+    "spec_version": "2.0",
+    "objects": [
+        {
+            "type": "identity",
+            "id": "identity--3532c56d-ea72-48be-a2ad-1a53f4c9c6d3",
+            "name": "Onelogin",
+            "identity_class": "events"
+        },
+        {
+            "id": "observed-data--17c193b8-6b7e-48c8-837d-b699e874d392",
+            "type": "observed-data",
+            "created_by_ref": "identity--3532c56d-ea72-48be-a2ad-1a53f4c9c6d3",
+            "created": "2021-08-02T06:06:52.127Z",
+            "modified": "2021-08-02T06:06:52.127Z",
+            "objects": {
+                "0": {
+                    "type": "x-onelogin-finding",
+                    "unique_id": 81004691744,
+                    "event_type_id": 149,
+                    "notes": "Default",
+                    "role_id": 441778,
+                    "app_id": "Default",
+                    "custom_message": "Default",
+                    "role_name": "Default",
+                    "app_name": "Default",
+                    "group_name": "Default",
+                    "otp_device_name": "Default",
+                    "operation_name": "Default",
+                    "directory_sync_run_id": "Default",
+                    "directory_id": 12345678,
+                    "resolution": "resolution",
+                    "client_id": 12345678,
+                    "resource_type_id": "Default",
+                    "browser_fingerprint": true
+                },
+                "1": {
+                    "type": "x-ibm-finding",
+                    "time_observed": "2021-06-22T13:12:06.437Z",
+                    "policy_id": 123,
+                    "name": "policy_name"
+                },
+                "2": {
+                    "type": "user-account",
+                    "x_account_id": 192204,
+                    "user_id": "138593517",
+                    "x_actor_user_id": 12345,
+                    "x_assuming_acting_user_id": 12345,
+                    "x_actor_user_name": "Mapping",
+                    "display_name": "Firstname Lastname"
+                },
+                "3": {
+                    "type": "ipv4-addr",
+                    "value": "121.0.0.1",
+                    "x_proxy_ip": "121.0.0.1"
```

Comment:
Does proxy_ip come in from OneLogin as a different value or would it always be the same as `ipaddr`? Since they come in as different fields, I think they should be treated as different STIX `ipv4-addr` objects. It doesn't make sense to have it as a custom property on this object when it may be a different IP, and if it's the same IP then it's just duplicated information.